installer/win/BuildWixSharpInstaller.bat

```diff
@@ -0,0 +1,2 @@
+set WIXSHARP_WIXDIR=.\WixSharpToolSet\Wix_bin\bin
+.\WixSharpToolSet\cscs Symphony.cs
```

Comment:
I think it'd be better to have a new line at the end of a file.

Reply:
It's a batch file, it doesn't care :)